packages/cosmic-swingset/Makefile

```diff
@@ -21,9 +21,9 @@ AGC_START_ARGS =
 
 BIN := $(shell echo $${GOBIN-$${GOPATH-$$HOME/go}/bin})
 
-all: build-chain install install-helper
+all: build-chain install install-agd install-helper
```

Comment:
Is `install-helper` supposed to stay around? It refers to `$(BIN)/ag-cosmos-helper`, which is no more, yes?

Reply:
> Is `install-helper` supposed to stay around? It refers to `$(BIN)/ag-cosmos-helper`, which is no more, yes?

It is deprecated but not retired.  Maybe I just didn't know what _superseded_ meant.